sdk/lib/rtl/critical.c

```diff
@@ -168,7 +168,8 @@ RtlpWaitForCriticalSection(PRTL_CRITICAL_SECTION CriticalSection)
             /* Is this the 2nd time we've timed out? */
             if (LastChance)
             {
-                ERROR_DBGBREAK("Deadlock: 0x%p\n", CriticalSection);
+                ERROR_DBGBREAK("(2/2) Deadlock: critical section %p, owned by thread %p\n",
```

Comment:
Please use `0x` in front of the `%p` so that a reader of a debug log doesn't have to guesscheck what a value like `47031250` could be.

Reply:
Done. (Tell me if you want that globally in this file.)

Reply:
It's fine (was just for the new dprints).